pkg/util/log/log_test.go

```diff
@@ -11,6 +11,41 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func TestMapStatusCodeToResultType(t *testing.T) {
```

Comment:
This is just a minor thing that may not be worth it, but we could remove the expectedResult (eg. "SuccessResultType") from the test name to not couple the test name with the implementation (if the name of the result type changes, the test name should not change, because the behavior is the same no matter if it is called "UserErrorResultType" or "ErrorFromTheUserResultType").